src/events/lavalink/rawLavalinkPlayerUpdate.ts

```diff
@@ -0,0 +1,19 @@
+import { Events } from '@lib/types/Enums';
+import { LavalinkPlayerUpdateEvent } from '@utils/LavalinkUtils';
+import { Event, EventStore } from 'klasa';
+
+export default class extends Event {
+
+	public constructor(store: EventStore, file: string[], directory: string) {
+		super(store, file, directory, {
+			emitter: store.client.lavalink!,
+			event: 'playerUpdate'
+		});
+	}
```

Comment:
Use `@ApplyOptions` please

Reply:
Not possible as it's accessing to the `store` variable.